utils/snapshot_extractor.mjs

```diff
@@ -0,0 +1,83 @@
+// @format
+import "dotenv/config";
```

Comment:
remove and ensure env vars via ava config or github actions

Reply:
We will have to add the env variables in github actions for sure. However, adding them to ava config will create trouble while commiting the files.

Reply:
I wanna replicate what I've done here: https://github.com/neume-network/core/blob/6aa74b519dc3b2ab55429f50311dc03aad7c0a24/package.json#L49 does that make sense?